rllib/examples/bandit/lin_ts_train_wheel_env.py

```diff
@@ -32,7 +32,12 @@ def plot_model_weights(means, covs):
 if __name__ == "__main__":
     num_iter = 10
     print("Running training for %s time steps" % num_iter)
-    trainer = BanditLinTSTrainer(env=WheelBanditEnv)
+    config = {
```

Comment:
Could we make this a command line arg, like in most other example scripts?

`--framework=[tf|tf2|torch]` and `--eager-tracing`.

You can copy paste this from any other example script, e.g. `rllib/examples/attention_nets.py`

Reply:
done